Makefile

```diff
@@ -55,6 +55,12 @@ debug:
 	docker build --no-cache -t aws-fluent-bit-plugins:latest -f Dockerfile.plugins .
 	docker build --no-cache -t amazon/aws-for-fluent-bit:debug -f Dockerfile.debug .
 
+.PHONY: debug-init
+debug:
+	docker build --no-cache -t aws-fluent-bit-plugins:latest -f Dockerfile.plugins .
```

Comment:
@matthewfala can you also update our debug targets for the step where we build the linux go plugins. That no longer works simply with the Dockerfile.plugins like we see here, you need to make the make debug targets depend on the `linux-plugins` target. Same as we do for `release` target.

Reply:
hmm yep.